docs/content/preview/deploy/multi-dc/async-replication.md

```diff
@@ -184,255 +182,264 @@ When both universes use different certificates, you need to store the certificat
     bootstrap_cdc_producer 000033e1000030008000000000004000,000033e1000030008000000000004003,000033e1000030008000000000004006
     ```
 
-    This command returns a list of bootstrap_ids, one per table id as below:
-    ```sh
+    The following output is a list of bootstrap IDs, one per table ID:
+    ```output
     table id: 000033e1000030008000000000004000, CDC bootstrap id: fb156717174941008e54fa958e613c10
     table id: 000033e1000030008000000000004003, CDC bootstrap id: a2a46f5cbf8446a3a5099b5ceeaac28b
     table id: 000033e1000030008000000000004006, CDC bootstrap id: c967967523eb4e03bcc201bb464e0679
     ```
 
-2. Take the backup of the tables on the source universe and restore at target universe. [Backup-Restore](../../../manage/backup-restore/)
-3. Then, set up the replication stream, using the bootstrap_ids generated in step 1
+2. Take the backup of the tables on the source universe and restore at the target universe by following instructions from [Backup and restore](../../../manage/backup-restore/).
+3. Execute the following command to set up the replication stream using the bootstrap IDs generated in step 1. Ensure that the bootstrap IDs are in the same order as their corresponding table IDs.
 
-  {{< note title="Note" >}}
-  It is important that the bootstrap_ids are in the same order as their corresponding table_ids!
-  {{< /note >}}
+      ```sh
+    ./bin/yb-admin -master_addresses <target_universe_master_addresses> setup_universe_replication \
+      <source_universe_uuid>_<replication_stream_name> <source_universe_master_addresses> \
+      <comma_separated_source_universe_table_ids> <comma_separated_bootstrap_ids>
+    ```
 
-  ```sh
-  ./bin/yb-admin -master_addresses <target_universe_master_addresses> setup_universe_replication \
-  <source_universe_uuid>_<replication_stream_name> <source_universe_master_addresses> \
-  <comma_separated_source_universe_table_ids> <comma_separated_bootstrap_ids>
-  ```
+    Consider the following example:
 
-  For Example:
-  ```sh
-  ./bin/yb-admin-master_addresses 127.0.0.11:7100,127.0.0.12:7100,127.0.0.13:7100 setup_universe_replication \
-  00000000-1111-2222-3333-444444444444_xCluster1 127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 \
-  000033e1000030008000000000004000,000033e1000030008000000000004003,000033e1000030008000000000004006 \
-  fb156717174941008e54fa958e613c10,a2a46f5cbf8446a3a5099b5ceeaac28b,c967967523eb4e03bcc201bb464e0679
-  ```
+      ```sh
+    ./bin/yb-admin-master_addresses 127.0.0.11:7100,127.0.0.12:7100,127.0.0.13:7100 setup_universe_replication \
+      00000000-1111-2222-3333-444444444444_xCluster1 127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 \
+      000033e1000030008000000000004000,000033e1000030008000000000004003,000033e1000030008000000000004006 \
+      fb156717174941008e54fa958e613c10,a2a46f5cbf8446a3a5099b5ceeaac28b,c967967523eb4e03bcc201bb464e0679
+      ```
+
+You can modify the bootstrap as follows:
```

Comment:
Should this be a subitem in the page outline?

Reply:
Added a section header